src/App.tsx

```diff
@@ -1,27 +1,113 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type Props = {};
+
+type State = {
+  isClockVisible: boolean,
+  x: number,
+  clockName: number,
 };
 
+class App extends React.Component <Props, State> {
+  state = {
+    x: 0,
+    isClockVisible: true,
+    clockName: 182,
+  };
+
+  componentDidMount() {
+    setInterval(() => {
+      this.setState((prevState) => ({ x: prevState.x + 1 }));
+    }, 1000);
+  }
```

Comment:
still don't get why do you need this `x`

Reply:
As mentioned above, to have a time counter of an App being mounted.
Should I remove it?

Reply:
Would be nice to, but no strong opinion